### ISSUE TITLE
[Dev Work] Rework LoadedComicInfo class

### ISSUE THREAD

**Issue by ThePromidius** [MangaManagerORG/Manga-Manager#144]
The focus of this story is to extract logic from LoadedComicInfo class.
With all the cover attributes, the class has become polluted.

Some possible ideas:
- Move logic outside using command design pattern (Would require a complete rework)
- Divide the class into subclases processing, metadata and cover. (procesign implements metadata and cover processing/functionality due to it being the one that actually saves to files)

While at it it'd be required to add more unit tests since it's the core of MangaManager.

To sum up, a todo list:

- [ ] Design a new structure that's clearer and easier to read
- [ ] Easy accesible for plugins
- [ ] Unit tests
- [ ] Small PRs at a time

**Comment by ThePromidius** [comment]
#159 tackles this a bit
more specific a48b6619c8c15bd99c17da022efb1e1009f177de
